includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2950,6 +2951,17 @@ public function append_stripelink_button( $field, $key, $args, $value ) {
 		return $field;
 	}
 
+
+	/**
+	 * Get the WC_Payments_Order from order id.
+	 *
+	 * @param  mixed $order the order_id or order object.
+	 * @return WC_Payments_Order order object with additional methods.
+	 */
+	public function wcpay_get_order( $order ) {
```

Comment:
I might see the answer to my question somewhere else in the PR, but why is this a method of the gateway class? It makes sense as a stand-alone function, or as something more easily accessible, ex. `WC_Payments::get_order()`. I prefer either just `get_order()`, or if we want to indicate it's a WCPay one, let's make it `get_wcpay_order()`.

Reply:
Removed.